vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -17,14 +18,15 @@ import {
 } from "@microsoft/applicationinsights-core-js";
 import { PageViewManager, IAppInsightsInternal } from "./Telemetry/PageViewManager";
 import { PageVisitTimeManager } from "./Telemetry/PageVisitTimeManager";
+import { PageViewPerformanceManager } from './Telemetry/PageViewPerformanceManager';
 import { ITelemetryConfig } from "../JavaScriptSDK.Interfaces/ITelemetryConfig";
 
 "use strict";
 
 const durationProperty: string = "duration";
 
 export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IAppInsightsInternal {
-    public static Version = "2.0.1-beta";
+    public static Version = "2.0.0-beta.3";
```

Comment:
this is unused, can be removed. As per discussion with @markwolff , we should use this to stamp the sdk.libVer in the overall sku code (currently properties plugin is doing the work, but that may not be avl. in all skus).

Reply:
This is being used in core, is being added to the event ext.sdk.libVer property, AI channel doesn't use this and first party properties override it but is actually being used